com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalDecalSubTarget.cs

```diff
@@ -74,8 +75,11 @@ public UniversalDecalSubTarget()
 
         public override bool IsActive() => true;
 
+        protected override ShaderUtils.ShaderID shaderID => ShaderUtils.ShaderID.SG_Decal;
+
         public override void Setup(ref TargetSetupContext context)
         {
+            base.Setup(ref context);
```

Comment:
N.B.: Decal Sub Target still uses an independent template [DecalPass.template
](https://github.com/Unity-Technologies/Graphics/blob/master/com.unity.render-pipelines.universal/Editor/Decal/DecalPass.template) and it doesn't include the `kUberTemplatePath`.
It probably deserves to be factorized with the common template at some point.